packages/react-radio/src/stories/RadioGroupLabeled.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { Label } from '@fluentui/react-label';
+import { useId } from '@fluentui/react-utilities';
+import { Radio, RadioGroup } from '../index';
+
+export const Labeled = () => {
+  const labelId = useId('label-');
+  return (
+    <>
+      <Label id={labelId} required>
+        Pick an option:
+      </Label>
+      <RadioGroup aria-labelledby={labelId}>
+        <Radio value="A" label="Option A" />
+        <Radio value="B" label="Option B" />
+        <Radio value="C" label="Option C" />
+        <Radio value="D" label="Option D" />
+      </RadioGroup>
+    </>
+  );
+};
+Labeled.parameters = {
+  docs: {
+    description: {
+      story:
+        'Use the `aria-labelledby` property on RadioGroup to associate a label with the group.<br />' +
+        '**Note**: The `<label>` attribute `for` (`htmlFor`) does _not_ work with RadioGroup.',
```

Comment:
is there an MDN reference we can't point to here?

Reply:
Not that I could find. Most of the documentation for radio groups says to use `<fieldset>` and `<legend>`, or as an alternative use `role="radiogroup"` and `aria-labelledby="..."`. 

Nothing I could find specifically addresses the lack of support for `<label for="...">`. I suppose maybe it's "obvious" because RadioGroup is itself not an input. We could leave it out, but I wanted warn people off a potential mistake because the rest of our input controls work with `<label for="...">`.

Reply:
Oh no, it's fine, and important. Just curious if there was a doc to point to.

Reply:
You could maybe call out that this is because `Radio` is an input but `RadioGroup` is not.

```suggestion
        '**Note**: The `<label>` attribute `for` (`htmlFor`) does _not_ work with RadioGroup because Radio is the input, not RadioGroup.',
```